ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -2877,6 +2877,15 @@ literals:
 - oil_pressure_sensor:
     flexible_min: 0.0
     flexible_max: 1034214.0
+- oil_tank_temperature_sensor:
```

Comment:
I don't think we need to specify `oil_tank` - please use existing `oil_temperature_sensor` and `oil_pressure_sensor`

Reply:
fixed